go/teams/chain.go

```diff
@@ -1518,18 +1530,50 @@ func (t *TeamSigChainPlayer) addInnerLink(
 							if err != nil {
 								return err
 							}
-							if _, err := prevState.GetLatestUVWithUID(uv.Uid); err != nil {
-								return err
+							addedUIDs[uv.Uid] = true
+							_, err = prevState.GetLatestUVWithUID(uv.Uid)
+							if err == nil {
```

Comment:
Do you need to handle the `err != nil` case?

Reply:
on error (when it can't find crypto member UV with given uid) it falls through to look for active invites for uid. the unified diff is a bit weird because govet told me to remove else branch and outdent everything